pkg/config/etcd.go

```diff
@@ -11,7 +11,7 @@ const (
 type EtcdConfig struct {
 	// Set a memory limit on the etcd process; etcd will begin paging
 	// memory when it gets to this value. 0 means no limit.
-	MemoryLimitMB uint64 `json:"memoryLimitMB"`
+	MemoryLimitMB uint64 `json:"memoryLimitMB" config:"comment,default=0"`
```

Comment:
I see kubebuilder directives to set defaults elsewhere, but this uses a different approach. What's the difference?

Reply:
Cleaned those up, we should be good to rely on just kubebuilder comments now.